### DISCUSSION TITLE
Dashboard didn't show me an inbox nor todo list

### DISCUSSION THREAD

**Discussion by mepi0011** [jonaswinkler/paperless-ng#1315] [Support] [answered]
The dashboard didn't show me an inbox nor todo list, like the picture [here](https://paperless-ng.readthedocs.io/en/latest/_images/dashboard.png)
Is this a special setting?

**Comment by TheFehr** [comment, marked as answer]
By saving a filtered view and then going to Settings -> Saved views you can decide which views you want to be shown on your dashboard. 

The system from the example picture will have a Inbox Tag and a saved view that filters for that Tag. As well as a TODO Tag and a corresponding filtered view for that Tag.

And both will have the "Show on dashboard" checkboxes checked in the Saved views Settings


I hope this helps

**Reply by mepi0011** [reply]
Thank you for the hint now it works.

I had not read that from the English welcome text (English is not my native language).
